Lesson26/merve2/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Image Gallery</title>
+  <link rel="stylesheet" type="text/css" href="./styles.css">
+</head>
+<body>
+  <h1>Image Gallery</h1>
+  <button id="fetch-button" class="button">Fetch Images</button>
+  <div id="image-container"></div>
```

Comment:
Move this div element below button and select element. Right now select and the button jumps on the screen every time images are fetched.